src/client/styles/components/FilterPopup.scss

```diff
@@ -1,12 +1,38 @@
-.nypl-full-width-wrapper.filter-page {
-  padding-bottom: 0;
+.form-full-width {
+  border-bottom: 1px solid;
+  border-top: 1px solid;
+
+  &.bottom-action-row {
+    border-bottom: none;
+  }
+
+  .nypl-full-width-wrapper {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
 
   .nypl-column-three-quarters {
-    margin-bottom: -1px;
+    margin-bottom: 0;
+  }
+}
+
+.nypl-full-width-wrapper {
+  &.filter-page {
+    padding-bottom: 0;
+
+    .nypl-column-three-quarters {
+      margin-bottom: -1px;
+    }
   }
 }
 
 .filter-container {
+  > .nypl-full-width-wrapper {
```

Comment:
Yes, it means that I want to target a direct child of `.filter-container`. There could be other `.nypl-full-width-wrapper` divs in `filter-container` but this will only target a direct child and not any grandchildren (which I don't want).

Reply:
nope.